### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by AlexXer**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
```
Comment (AlexXer) [inline index.html:1]: https://www.w3schools.com/HTML/html_responsive.asp

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">   	
+    <title>Document</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+</head>
+<body>
+   <div class="mainContainer">
+    <div class="headerMenuBlock">
```
Comment (AlexXer) [inline index.html]: header > nav (semantic html)

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+    .container_images .loading_arrow2{
+        height: 550px;
+        padding-top: 150px;
+        font-size: 100px
+    }
+    .logo{
+        margin-left: 18px;
+        color: #ebebeb;
+    }
+    .miniMenu{
```
Comment (AlexXer) [inline styles.css]: to read: Yandex BEM

File: index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            			<li>PSD WEBSITES</li>
+            			<li>UL KITS</li>
+            			<li>PREMIUM PSD FILES</li>
+            			<li>ABOUT</li>
+            			<li>BLOG</li> 
+        			</ul>
+        		</nav>
+        		<nav class="mini_navigation">
+            		<div class="logo">Navigation</div>
+            		<div class="miniMenu">&#8801</div>
```
Comment (aliaksandrkrasnou) [inline index.html]: ; is missing

**Comment by aliaksandrkrasnou** [comment]
Approved, okay 👍

**Pull request closed without merging**